package/crossplane.yaml

```diff
@@ -2,3 +2,10 @@ apiVersion: meta.pkg.crossplane.io/v1
 kind: Provider
 metadata:
   name: provider-ksyun
+  annotations:
+    meta.crossplane.io/maintainer: <ADD EMAIL>
+    meta.crossplane.io/source: github.com/notone0010/provider-ksyun
+    meta.crossplane.io/description: |
+      A Crossplane provider that exposes managed resources for the Ksyun API.
+    meta.crossplane.io/readme: |
+      <ADD README>
```

Comment:
@notone0010 feel free to change the metadata here, which will render in the Upbound listing. all object annotations described in the [xpkg spec](https://github.com/crossplane/crossplane/blob/master/contributing/specifications/xpkg.md#object-annotations) will be honoured.

Reply:
@jastang I have merged and completed the readme.

Reply:
And next, what need I do for the package publish?

Reply:
@notone0010 you'll need to add Github Actions Secrets to this repository. See the keys in ci.yml here:

https://github.com/notone0010/provider-ksyun/actions/runs/6610514274/workflow#L264-L265

That can be either be a [robot](https://docs.upbound.io/concepts/accounts/robots/) token/id pair, or your own personal credentials.

Reply:
> @notone0010 you'll need to add Github Actions Secrets to this repository. See the keys in ci.yml here:
> 
> https://github.com/notone0010/provider-ksyun/actions/runs/6610514274/workflow#L264-L265
> 
> That can be either be a [robot](https://docs.upbound.io/concepts/accounts/robots/) token/id pair, or your own personal credentials.

@jastang I still encountered an authentication error when I set Github Actions Secrets. 

The Github Actions Secrets settings' screenshot
<img width="843" alt="image" src="https://github.com/notone0010/provider-ksyun/assets/42347556/cf9cb89b-ff45-41f3-b206-85371a63c350">

Error details
<img width="1036" alt="image" src="https://github.com/notone0010/provider-ksyun/assets/42347556/f56aeab5-4c50-44ae-baec-6a14dee9c793">

Reply:
This specific workflow is not well-documented at the moment, but the recommended path is to use a robot token. 

You'll need to assign your robot to a team, and give that team WRITE permissions to `noteone/provider-ksyun` repo. See https://docs.upbound.io/concepts/accounts/teams/#manage-repository-permissions.

If we're still running into issues, let's open an issue to track this so we don't get lost in the merged PR 👍🏼